packages/admin/admin/src/common/Tooltip.tsx

```diff
@@ -185,26 +180,10 @@ const TooltipPopper = createComponentSlot(MuiPopper)<TooltipClassKey, OwnerState
 );
 
 export const Tooltip = (inProps: TooltipProps) => {
-    const {
-        trigger = "hover",
-        variant = "dark",
-        disableInteractive,
-        arrow,
-        children,
-        ...props
-    } = useThemeProps({ props: inProps, name: "CometAdminTooltip" });
+    const { variant = "dark", disableInteractive, arrow, children, ...props } = useThemeProps({ props: inProps, name: "CometAdminTooltip" });
     const theme = useTheme();
 
-    const [open, setOpen] = useState(false);
-
-    const handleTooltipClose = () => {
-        setOpen(false);
-    };
-
-    const toggleTooltip = (event: MouseEvent) => {
-        event.stopPropagation();
-        setOpen(!open);
-    };
+    const [open] = useState(false);
```

Comment:
A state that never changes should not be a state.
Is this value actually needed for anything?